ChangeLog.md

```diff
@@ -2,6 +2,18 @@
 
 ## New Features / Critical Changes
 
+- *Topology Package*
+
+ - Adding the half-edge data structure to represent arbitrary
+   two-dimensional combinatorial surfaces with or without boundary
+   (Jacques-Olivier Lachaud
+    [#1266](https://github.com/DGtal-team/DGtal/pull/1266))
+
+- *Shapes Package*
+ - Adding classes and helpers to create triangulated surfaces and
+   to convert them from/to mesh and digital surfaces (Jacques-Olivier
```

Comment:
Agree but Mesh -> digital surfaces is not a feature of this PR I think.. Maybe minor rephrasing?